account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -14,10 +20,16 @@
 import { local070Instance } from "~test/instances.js";
 import { setBalance } from "viem/actions";
 import { accounts } from "~test/constants.js";
-import { getDefaultSingleSignerValidationModuleAddress } from "../modules/utils.js";
+import {
+  getDefaultSingleSignerValidationModuleAddress,
+  getDefaultAllowlistModuleAddress,
+  getDefaultNativeTokenLimitModuleAddress,
+} from "../modules/utils.js";
 import { SingleSignerValidationModule } from "../modules/single-signer-validation/module.js";
 import { allowlistModule } from "../modules/allowlist-module/module.js";
 import { HookType } from "../actions/common/types.js";
+import { nativeTokenLimitModule } from "../modules/native-token-limit-module/module.js";
+import { nativeTokenLimitModuleAbi } from "../modules/native-token-limit-module/abis/nativeTokenLimitModuleAbi.js";
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'nativeTokenLimitModuleAbi' is defined but never used.
<!-- __reviewdog__:ChA2NzE5ODJlNjMzZmUyNzU2EgZyZGpzb24= -->